src/MercadoPago/Client/Order/OrderTransactionClient.php

```diff
@@ -59,4 +60,23 @@ public function update(string $order_id, string $transaction_id, array $request,
         $result->setResponse($response);
         return $result;
     }
+
+    /**
+     * Method responsible for deleting transactions for an Order.
+     * @param string $order_id Order ID.
+     * @param string $transaction_id Transaction ID.
+     * @param \MercadoPago\Client\Common\RequestOptions request options to be sent.
+     * @return Response
+     * @throws \MercadoPago\Exceptions\MPApiException if the request fails.
+     * @throws \Exception if the request fails.
+     */
+    public function deleteTransaction(string $order_id, string $transaction_id, ?RequestOptions $request_options = null): MPResponse
```

Comment:
Já está client especifico de transaction
```suggestion
    public function delete(string $order_id, string $transaction_id, ?RequestOptions $request_options = null): MPResponse
```